src/flame-chart.ts

```diff
@@ -34,7 +35,7 @@ export type FlameChartOptions = {
 
 const defaultSettings: FlameChartSettings = {};
 
-export default class FlameChart extends FlameChartContainer<FlameChartStyles> {
+export class FlameChart extends FlameChartContainer<FlameChartStyles> {
```

Comment:
named export and default export of the same thing?
not sure how to feel about that

Reply:
I can understand how it looks odd - but it makes for a nicer typescript api (IMHO). It is backwards compat, non-breaking.

I have a lot more chagnes pending - converting the example to typescript - and the code looks cleaner without using the default export.